tests/integration/test_interface.py

```diff
@@ -27,46 +29,60 @@ def test_generate_decennial_census(
     assert noised_data.equals(noised_data_same_seed)
     assert not noised_data.equals(noised_data_different_seed)
     assert not data.equals(noised_data)
-    # TODO: Confirm correct columns exist once the interface functions
-    # modify them
-    # TODO: if we sort out dtype schemas
-    # for col in noised_data.columns:
-    # assert data[col].dtype == noised_data[col].dtype
-    # TODO: Iterate through cols and check that the percentage of errors makes sense
-    # eg, if 25% typographic error and 1% OCR
-    # 1. Use a default config file
-    # 2.
-
-    config = get_configuration(dummy_config)["decennial_census"]
-
-    # Confirm omission and duplication seems reasonable
-    # TODO: when omission function gets implemented.
-    orig_idx = data.index
-    noised_idx = noised_data.index
-    # assert np.isclose(len(set(orig_idx) - set(noised_idx)) / len(data), config.omission)
-    # TODO: when duplication function gets implemented
-    # assert np.isclose(noised_data.duplicated().sum() / len(data), config.duplication)
-
-    # Check that column-level noise seem reasonable
-    # NOTE: this is not perfect because (1) it is only looking at row-level
-    # noise and not token-based noise and (2) it is not accounting for the
-    # fact that noising can occur on duplicated rows which have been removed
-    # for comparison purposes.
-    common_idx = set(orig_idx).intersection(set(noised_idx))
-    common_data = data.loc[common_idx]
-    common_noised_data = noised_data.loc[common_idx].drop_duplicates()
-    assert common_data.shape == common_noised_data.shape
+    assert set(noised_data.columns) == set(data.columns)
+
+
+def test_noise_decennial_census_with_two_noise_functions(dummy_census_data, tmp_path_factory):
+    # todo: Make config tree with 2 function calls
+    # Make simple config tree to test 2 noise functions work together
+    config_tree = ConfigTree(
+        {
+            "decennial_census": {
+                "first_name": {
+                    "missing_data": {"row_noise_level": 0.01},
+                },
+                "state": {
+                    "missing_data": {"row_noise_level": 0.01},
```

Comment:
Have we discussed yet which permutations of functions we need for good-enough coverage? This one is obvious b/c incorrect selection is the second function to be implemented, but what about from here on out? @rmudambi

Reply:
We haven't.  This is also something I was wondering.  We could in theory try to set this up to be more flexible and take additional args that create a config tree from args so we could have several permutations of two tests which would add a lot more coverage but I was unsure how to do that.